db/migrate/20160222091312_remove_other_accreditation.rb

```diff
@@ -0,0 +1,12 @@
+class RemoveOtherAccreditation < ActiveRecord::Migration
+  class Accreditation < ActiveRecord::Base
+  end
+
+  def up
+    Accreditation.where(name: 'Other').destroy_all
+  end
+
+  def down
+    fail ActiveRecord::IrreversibleMigration
```

Comment:
Possibly. The problem with adding it back in is that all of the other records related to it are hardcoded to the ID ('5'). So adding it back in would probably make it have an ID of 6, and nothing would work anyway.

Is it possible not to have a `down` state? I'm happy to leave that off but not sure of the consequences.